docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,91 @@
+----
++
+Users need to explicitly set security contexts in their manifests to avoid these warnings.
++
+Red Hat plans to switch Pod Security Admission to restricted enforcement globally in a future minor release.
+When restricted enforcement will be enabled, pods with pod security violations will be rejected.
+
+vSphere CSI Topology generally available::
+
+This release of OpenShift provides support for defining regions and zones in vSphere through categories and tags.
+After the categories and tags are created, they can be referenced in MachineSets to create VMs in these failure domains.
++
+See the https://docs.openshift.com/container-platform/4.12/storage/container_storage_interface/persistent-storage-csi-vsphere.html#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[upstream documentation] for more details.
++
+NOTE: OpenShift 4.12 still doesn't support installation across multiple vSphere Datacenters.
+
+Support for CoreDNS cache tuning::
+
+You can now configure the time-to-live (TTL) duration of both successful and unsuccessful DNS queries cached by CoreDNS.
+
+'''
+
+Both the release notes and the blog post highlight a number of new operators which are available for OpenShift 4.12:
+
+Security Profiles Operator -- https://cloud.redhat.com/blog/whats-new-in-red-hat-openshift-4.12-blog#securityprofiles[What's New blog post]::
+
+The Security Profiles Operator (SPO) exposes the power of SELinux, seccomp, and AppArmor to end users.
+The technologies are not mutually exclusive, and these are combined in the SPO to make it easier to consume.
+Seccomp reduces the chance that a kernel vulnerability will be successfully exploited, whereas SELinux and AppArmor prevent applications from accessing files they should not.
++
+See the https://docs.openshift.com/container-platform/4.12/security/security_profiles_operator/spo-overview.html[upstream documentation] for more details.
+
+Ingress Node Firewall Operator -- https://cloud.redhat.com/blog/a-guide-to-providing-insight-with-network-observability[What's New blog post]::
+
+OpenShift 4.12 introduces a new stateless Ingress Node Firewall Operator.
+Administrators configure firewall rules at the node level to control the flow of network traffic to/from the node by controlling which interface and remote hosts the Kubernetes API server can be accessed from.
++
+See the https://docs.openshift.com/container-platform/4.12/networking/ingress-node-firewall-operator.html#ingress-node-firewall-operator[upstream documentation] for more details.
+
+Network Observability Operator -- https://cloud.redhat.com/blog/whats-new-in-red-hat-openshift-4.12-blog#observabilityoperator[What's New blog post]::
```

Comment:
Not sure we want to mention this -- this looks like a direct competitor to Cilium's observability features